webapp/portlet/src/main/webapp/idm-groups-management/components/GroupsManagementFormDrawer.vue

```diff
@@ -186,9 +195,8 @@ export default {
       }).then(resp => {
         if (!resp || !resp.ok) {
           if (resp.status === 400) {
-            return resp.text().then(error => {
-              this.fieldError = error;
-              throw new Error(error);
+            return resp.text().then(() => {
+              throw new Error(this.$t('GroupsManagement.popup.message'));
```

Comment:
This change will affect the other errors to not display properly

Reply:
ok done